Cubical/Data/Sigma/Properties.agda

```diff
@@ -171,7 +178,40 @@ leftInv (Σ-cong-iso-fst {A = A} {B = B} isom) (x , y) = ΣPathP (leftInv isom x
       ∙∙ substRefl {B = B} y
 
 Σ-cong-equiv-fst : (e : A ≃ A') → Σ A (B ∘ equivFun e) ≃ Σ A' B
-Σ-cong-equiv-fst e = isoToEquiv (Σ-cong-iso-fst (equivToIso e))
+Σ-cong-equiv-fst {A = A} {A' = A'} {B = B} e = intro , isEqIntro
```

Comment:
If anybody knows a better way that doesn't involve writing out these long proofs by hand, let me know. At least it's direct and doesn't convert back and forth between `Iso`.

Reply:
What is the motivation for not having the shorter proof?

Reply:
If it's efficiency then please keep the shorter proof in a comment explaining this. Otherwise someone might come along and shorten it again in the future